### PR TITLE
[Azure.Monitor.Inestion] Update CHANGELOG.md for GA Release

### PR DESCRIPTION
**Pull request by nisha-bhatia**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,10 @@
 - Removed Model `UploadLogsError` representing the error and the associated logs that failed when uploading a subset of logs to Azure Monitor.
 - Removed Model `UploadLogsStatus` indicating the status of a logs upload operation.
 
+## 1.1.0 (2022-10-11)
+## Features Added
+- GA release with sovereign support
```
Comment (pallavit) [inline sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md:26]: Consider updating this to something like

Added multi-cloud support via `LogsIngestionAudience` to allow users to select the Azure cloud where the resource is located.

**Review by JoshLove-msft — APPROVED** [review]

**Review by pallavit — APPROVED** [review]

File: sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md [hunk 2]
```diff
@@ -21,6 +21,10 @@
 - Removed Model `UploadLogsError` representing the error and the associated logs that failed when uploading a subset of logs to Azure Monitor.
 - Removed Model `UploadLogsStatus` indicating the status of a logs upload operation.
 
+## 1.1.0 (2022-10-11)
```
Comment (srnagar) [inline sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md:24]: Fix the date of release and this should be at the top of the changelog as this is the most recent release.

Comment (nisha-bhatia) [reply sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md]: updated!

File: sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,14 @@
 # Release History
 
+## 1.1.0 (2022-10-10)
```
Comment (srnagar) [inline sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md:3]: Date should be 2023-10-10

**Review by srnagar — CHANGES_REQUESTED** [review]

File: sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md [hunk 4]
```diff
@@ -1,9 +1,14 @@
...
 
+## 1.1.0 (2023-10-10)
+
+## Features Added
+- Added `LogsIngestionAudience` for multi-cloud support to allow users to select the Azure cloud where the resource is located.
+
 ## 1.1.0-beta.1 (2023-10-10)
 
 ### Bugs Fixed
-  - Fix sovereign support for US Gov and China clouds
```
Comment (JoshLove-msft) [inline sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md:6]: Did we actually ship a beta with this? If so, don't we need to update the csproj now?

Comment (nisha-bhatia) [reply sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md:6]: updated - yes the beta was shipped

**Review by srnagar — APPROVED** [review]
LGTM

**Pull request merged**